htdocs/societe/list.php

```diff
@@ -1165,6 +1190,15 @@
 		print '</td>';
 		if (!$i) $totalarray['nbfield']++;
 	}
+	// Multiprice level
+	if (!empty($conf->global->PRODUIT_MULTIPRICES) || !empty($conf->global->PRODUIT_CUSTOMER_PRICES_BY_QTY_MULTIPRICES))
```

Comment:
The test here should be useless. Because if false, $arrayfields['s.price_level']['checked'] will always be empty. Right ?